next.config.mjs

```diff
@@ -1,4 +1,70 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
```

Comment:
why'd you add this file? add comments for what you're using here (we don't need any of this in what I have up and running so far, and want to avoid as much unnecessary c/p as possible)

Reply:
I added it to get the build working but only need the `images` key to allow contentstack images